xml/System.Net/IPEndPoint.xml

```diff
@@ -764,6 +806,44 @@
         </remarks>
       </Docs>
     </Member>
+    <MemberGroup MemberName="TryParse">
+      <AssemblyInfo>
```

Comment:
Why did you add `AssemblyInfo` items manually? These items are supposed to be auto-generated by the Docs repo.

Reply:
Same question about the other `AssemblyInfo` items in other sections of the PR.

Reply:
Because this is a `MemberGroup` which is not auto-generated. I also did it based on suggested example in https://github.com/dotnet/dotnet-api-docs/pull/2663#discussion_r298781159, which adds the `MemberGroup` node with an `AssemblyInfo` inside. 

But I think you are right, I removed the `AssemblyInfo` nodes and I was able to compile locally without problems, however I would like to have validation from @mairaw or @rpetrusha about this.

Reply:
Pinging @joelmartinez on this one. I also added this manually to other member groups just because others I saw had it.

Joel, is it okay to remove `AssemblyInfo` from the MemberGroup elements?

Reply:
@mairaw, yes, that is fine ... the membergroup, from a metadata perspective, is only informative; so it doesn't really conform to any of the other requirements reserved for members :)